ui/.eslintrc.yml

```diff
@@ -0,0 +1,6 @@
+extends: airbnb-base
+plugins:
+    - import
+rules:
+    no-underscore-dangle: [2, {allowAfterThis: true}]
+    linebreak-style: [error, windows]
```

Comment:
I'm not sure if this `linebreak-style: windows ` would give Linux users troubles.

Reply:
shall i remove this rule?

Reply:
@rehrumesh @malithsen WDYT? I'm not very familiar with ESLint